packages/mdc-side-sheet/README.md

```diff
@@ -1,4 +1,318 @@
-# Side Sheet
+<!--docs:
+title: "Side Sheets"
+layout: detail
+section: components
+excerpt: "Permanent, dismissible, and modal side sheets."
+iconId: side_navigation
+path: /catalog/side-sheets/
+-->
 
-The [Side Sheet component](https://material.io/go/design-sheets-side) is yet to be completed, please follow the [tracking issue](https://github.com/material-components/material-components-web/issues/2665) for more information.
+# Side Sheets
 
+The MDC Side Sheet is used to organize access to destinations and other functionality on an app.
```

Comment:
Based on material.io wording:

The MDC Side Sheet is a supplementary surface primarily used on tablet and desktop.